cmd/init.go

```diff
@@ -31,6 +31,7 @@ you started.
 
 func init() {
 	initCmd.Flags().BoolVarP(&cfg.Flags.Quickstart, "quickstart", "", false, `use defaults to setup KubeFox for quickstart guide`)
+	initCmd.Flags().BoolVarP(&cfg.Flags.RemoteRegistry, "remote-registry", "", false, `setup a remote container registry for the quickstart`)
```

Comment:
This confused me for a sec, I thought it was meant to pass the registry URL. Perhaps we change it to `setup-registry` instead?